app/containers/Forms/EmissionGasFields.tsx

```diff
@@ -48,9 +48,16 @@ const EmissionGasFields: React.FunctionComponent<FieldProps> = ({
           {formData.gasType} <br />
           <Col
             md={10}
-            style={{margin: 0, padding: 0, color: '#888', lineHeight: '17px'}}
+            style={{
+              margin: 0,
+              padding: 0,
+              color: '#797676',
+              lineHeight: '17px'
+            }}
           >
-            <small>{formData.gasDescription}</small>
+            <label htmlFor={`${idSchema?.$id}_annualEmission`}>
+              <small>{formData.gasDescription}</small>
+            </label>
```

Comment:
Rule: all inputs must have a label.

The content of `<small>` labels a corresponding `<input>`, so it's wrapped with a `<label>` element accordingly. 

(Side note: `htmlFor` is one of those React-specific attribute names - like `className` - to avoid clashing with reserved JS keywords. In plain HTML, this would be a `for` attribute like so: `<label for="id-of-input-this-labels">`.